betterdiscordctl

```diff
@@ -286,23 +288,31 @@ bdc_upgrade() {
 
 bdc_main() {
   if [[ -z $modules ]]; then
-    if [[ $snap ]]; then bdc_snap
-    elif [[ $flatpak ]]; then bdc_flatpak
+    if [[ $flatpak ]]; then bdc_flatpak
+    elif [[ $snap ]]; then bdc_snap
     else bdc_scan; fi
   else
     [[ -d $modules ]] || die 'ERROR: Discord modules directory not found.'
-    : "${modules%/*/modules}"; flavor=${_##*/discord}
+    flavor=${modules%/*/modules}
+    flavor=${flavor##*/discord}
   fi
+  [[ -d $modules ]] || die 'ERROR: Discord modules directory not found.' \
+      'Try specifying it with "--modules".'
   core=$modules/discord_desktop_core
   [[ -d $core ]] || die "ERROR: Directory 'discord_desktop_core' not found in $modules"
+  bd_config=$xdg_config/BetterDiscord
+  bd_asar=$bd_config/data/betterdiscord.asar
```

Comment:
There's no reason for this to be defined here.

Reply:
Oh yeah, `xdg_config` is never reassigned, so assigning it up in `# Variables` is fine.